pkg/k8sutil/k8sutil.go

```diff
@@ -752,6 +761,12 @@ func (k *K8sutil) CreateClientMasterDeployment(deploymentType, baseImage string,
 			},
 		}
 
+		// disable secrets volume mount when ssl is not enabled
```

Comment:
@stevesloka I'm not really happy with this part of the code. Any thoughts are welcome.